src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -24,17 +32,13 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\samples\\Azure.Network.Management.Interface\\Generated"
     },
-    "Azure.Storage.Tables": {
-      "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\samples\\Azure.Storage.Tables\\Generated"
-    },
-    "azure-parameter-grouping": {
+    "Azure.ResourceManager.Compute": {
       "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestServerProjects\\azure-parameter-grouping\\Generated"
+      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\Azure.ResourceManager.Compute\\Generated"
```

Comment:
We usually don't put actual service swaggers under `TestProjects`. `TestProjects` are hand-crafted swagger files to illustrate particular features and write tests against them. I recommend samples directory.